src/EditorFeatures/TestUtilities/TestExperimentationService.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Experiments;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    [Shared]
+    [Export(typeof(MockExperimentationService))]
```

Comment:
just curious, why does it need both Export and ExportWorkspaceService?
And will the PartNotDiscoverable only apply to ExportWorkspaceService?

Reply:
> why does it need both Export and ExportWorkspaceService?
I guess the `Export` it's not required, but it saves me a conversion :)

 >  will the PartNotDiscoverable only apply to ExportWorkspaceService
I believe so.